packages/sarif-to-markdown/src/sarif-to-markdown.ts

```diff
@@ -229,7 +229,7 @@ export const sarifToMarkdown = (options: sarifFormatterOptions): ((sarifLog: Log
             If pass the scan, results is empty array
             */
             const results =
-                run.results && run.results.length > 0
+                run.results && run.results.length > 0 && Object.keys(filteredResults).length > 0
```

Comment:
We can defined a variable instead of repeating the statment.
```js
const hasMessage = run.results && run.results.length > 0 && Object.keys(filteredResults).length > 0;
// or 
const hasResult = run.results && run.results.length > 0 && Object.keys(filteredResults).length > 0;
```

Reply:
Just pushed a commit with the constant.
There was a difference though, with these 2 conditions:
`run.results && run.results.length > 0 && Object.keys(filteredResults).length > 0`
vs
`run.results?.length !== 0 && Object.keys(filteredResults).length !== 0`